fpga/src/main/scala/arty/Configs.scala

```diff
@@ -24,7 +22,6 @@ class WithDefaultPeripherals extends Config((site, here, up) => {
     idcodePartNum = 0x000,
     idcodeManufId = 0x489,
     debugIdleCycles = 5)
-  case SerialTLKey => None // remove serialized tl port
```

Comment:
I think we should leave this as the default, then we re-add this in the "Sim" config. By default, most people will probably just go directly to building the bitstream and ignore simulation.

Reply:
Thanks and agreed—fixed in most recent commit.